packages/core/src/command/hl7v2-subscriptions/types.ts

```diff
@@ -9,16 +9,32 @@ export type SftpConfig = {
   remotePath: string;
 };
 
-export type HieConfig = {
-  name: string;
-  sftpConfig?: SftpConfig;
-  schema: Record<string, string>;
+export const addressFields = ["addressLine1", "addressLine2", "city", "state", "zip"] as const;
+export type AddressField = (typeof addressFields)[number];
+export type HieAddressFieldMapping = {
+  [K in AddressField]: string;
+};
+
+export type HieFieldMapping = {
+  scrambledId: string;
+  firstName: string;
+  lastName: string;
+  dob: string;
+  genderAtBirth: string;
+  ssn?: string;
+  phone?: string;
+  email?: string;
+  driversLicense?: string;
+  address: HieAddressFieldMapping[];
 };
```

Comment:
To reduce ambiguity lets call this MetriportToHieFieldMapping, HieFieldMapping could just as easily mean going in the other direction; hie -> metriport.